xbmc/GUIInfoManager.cpp

```diff
@@ -332,6 +337,37 @@ const infomap integer_bools[] =  {{ "isequal",          INTEGER_IS_EQUAL },
 ///     Returns true if pvr channel preview is active (used channel tag different
 ///     from played tag)
 ///   }
+///   \table_row3{   <b>`Player.TempoEnabled`</b>,
+///                  \anchor Player_TempoEnabled
+///                  _boolean_,
+///     Returns true if player supports tempo (i.e. speed up/down normal playback speed)
+///   }
+///   \table_row3{   <b>`Player.IsTempo`</b>,
+///                  \anchor Player_IsTempo
+///                  _boolean_,
+///     Returns true if player has tempo (i.e. is playing with a playback speed higher or
+///     lower than normal playback speed)
+///   }
+///   \table_row3{   <b>`Player.PlaySpeed`</b>,
+///                  \anchor Player_PlaySpeed
+///                  _string_,
+///     Returns the player playback speed (tempo) with the format %1.2f (1.00 means normal 
```

Comment:
this is not entirely correct. if returned values is != 1 and IsTempo is false, player goes ff/rw